clusterloader2/api/types.go

```diff
@@ -31,9 +31,6 @@ type TestScenario struct {
 	Identifier string `json:"identifier"`
 	// ConfigPath defines path to the file containing a single Config definition.
 	ConfigPath string `json:"configPath"`
-	// OverridePaths defines what override files should be applied
-	// to the config specified by the ConfigPath.
-	OverridePaths []string `json:"overridePaths"`
```

Comment:
Wait - why this was removed. How can I pass it now?
What happens with this one:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/experimental/storage/pod-startup/suite.yaml#L3

Reply:
Good point, we shouldn't have removed OverridePaths from TestScenario. Let me revert the change, we can roll-forward with the fix, which will be simply leaving this field as is.

Reply:
Oh I thought it would be passed as a global https://github.com/kubernetes/perf-tests/pull/1680/commits/4cf2c18e872d6bf18122717b66ac1787f9320744#diff-d6057f57c97c93f50c6e310f19970261bbd5d145d2f9682473e3749afe7abb97R119

But I didn't realize it could be scenario specific for test suites

Reply:
Yeah, my bad, I didn't explain it clear enough. The one passed via a flag is global, but it's also possible to do it per test scenario. I wanted to move only the global one

Reply:
Ah okay then I'll raise another PR later to revert the revert and apply a fix for suites